### PR TITLE
feat: add documents for e2e tests

### LINKED ISSUE #212: Add instructions to run E2E test
Explain and add make commands to facilitate E2E test

### PR DESCRIPTION
**Pull request by cool-develope**
Closes #212 

### Reviewers

Main reviewers:

- @arnaubennassar 
- @ARR552

### CODE REVIEW

File: docs/running_local.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+> WARNING: This documentation is outdated, it will be updated soon
```
Comment (ARR552) [inline docs/running_local.md:1]: Is it outdated?

**Review by ARR552 — APPROVED** [review]

File: docs/running_local.md (hunk 1 above)
Comment (cool-develope) [reply docs/running_local.md:1]: not now, let's keep this warning until prepare the full document

**Review by arnaubennassar — CHANGES_REQUESTED** [review]
Great job! The only thing I'm missing is to reference bot the diagram and the running doc from the main README

File: README.md [hunk 2]
```diff
@@ -5,6 +5,16 @@ to interact with the [bridge smart contract](https://github.com/0xPolygonHermez/
 
 The software is currently not ready to be used, and more instructions for running and developing it will be provided in the very near future.
```
Comment (arnaubennassar) [inline README.md:6]: remove this

**Review by arnaubennassar — CHANGES_REQUESTED** [review]

**Review by arnaubennassar — APPROVED** [review]

**Pull request merged**